CHANGELOG.next.asciidoc

```diff
@@ -35,6 +35,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Removed beats central management {pull}25696[25696], {issue}23908[23908]
 - MacOSX minimum supported version set to 10.14 {issue}24193{24193}
 - Add daemonset.name in pods controlled by DaemonSets {pull}26808[26808], {issue}25816[25816]
+- Refactor DiscoverKubernetesNode to add error handling and tests  {pull}26947[26947]
```

Comment:
I think you don't need changelog for this change

Reply:
Well, a changelog is needed, but for the user-facing change, something like "`NODE_NAME` environment variable is used now to discover the local kubernetes node name".

Reply:
And/or "Kubernetes autodiscover fails to start now if it cannot discover the local node name and scope is set to node".